docs/io.cozy.files_metadata.md

```diff
@@ -0,0 +1,391 @@
+
+# Metadata on `io.cozy.files` documents
+
+## Image files
+
+For pictures files, like `jpg`, `png`, `gif`...
+
+- `width`: {number}
+- `height`: {number}
+- `datetime`: {date} : date in original image file metadata
+- `gps`: {map} : localization metadata with the following attributes
+    - `lat`: {float}: latitude
+    - `long`: {float}: longitude
+    - `city`: {string}: nearest city (optional)
+    - `zip`: {string}: postal code of the nearest city (optional)
+    - `country`: {string}: name of the associated country if any (optional)
+- `persons`: {array}: the maps can have the following attributes (optional)
+    - `name`: {string}: then name of the tagged person on the photo
+    - `created_at`: {date}: date of creation of the tag
+    - `x`: {float}: x coordinate in the photo where the person is
+    - `y`: {float}: y coordinate in the photo where the person is
+
+
+## Administrative documents
+
+Mostly imported by the connectors, cozy stores many different types of administrative documents, like invoices, contracts or certificates.
+
+### Data structure
+
+#### Common attributes
+There attributes are share by all the documents types covered by the specification.
```

Comment:
These attributes are shared by all the document types

Reply:
ok, thanks